style.css

```diff
@@ -173,12 +174,56 @@ ul {
   margin-bottom: 20%;
 }
 
-.Works {
+.cards-container {
   margin-left: 10%;
   margin-right: 10%;
   display: grid;
 }
 
+.modal-wrapper {
+  width: 100vw;
+  height: 78vh;
```

Comment:
- [ ] - [ OPTIONAL ] Well done on creating the pop-up, it is working fine but the design is a bit broken in my opinion, please take a look at this record, currently, the user is even able to open 2 pop-ups in one time, and the window is following the page on scroll, a better way is to fix it and keep it centered in the middle of the page.

- [ ] this is what your pop-up design should look like

<img width="664" alt="1" src="https://user-images.githubusercontent.com/77843050/208974679-8ce661de-96a3-42a5-af3d-583f467d7428.png">

- [ ] This is how yours is looking like, I would suggest you make a few touches on the style again before we jump on the next step

![5](https://user-images.githubusercontent.com/77843050/208973268-8b00aa93-ac2f-4e33-adca-d908bdc50834.gif)

Reply:
dear @joskalenda , I used js to solve the problem, before opening a new window, I close all of them, does this a good way? because I tried changing the position to fix it and etc, it makes pop windows don't come over everything. only absolute workes in this situation